SYCL/FilterSelector/select.cpp

```diff
@@ -6,6 +6,7 @@
 //
 // Failing on HIP AMD
 // XFAIL: hip_amd
+// UNSUPPORTED: esimd_emulator
```

Comment:
If I right understand, the test just checks the parameters of the esimd_emulator device and doesn't run any kernels. Should we disable the test for `esimd_emulator`?